scripts/gen_coverage.py

```diff
@@ -146,6 +143,7 @@ def find_objects():
                 "export",
                 "-format=lcov",
                 "-ignore-filename-regex=/tmp/icpx*",
+                r"-ignore-filename-regex=.*/backend/kernels/elementwise_functions/.*\.hpp$",
```

Comment:
there are also kernels in `extensions` which will have to be ignored

Reply:
I'll do that in the follow up PR, because they have to be separated to dedicated headers first. Otherwise it's impossible to ignore.